bs-react-native-next/src/apis/Animated.md

```diff
@@ -243,11 +243,12 @@ external parallel:
 [@bs.module "react-native"] [@bs.scope "Animated"]
 external stagger: (float, array(Animation.t)) => Animation.t = "";
 
-[@bs.module "react-native"] [@bs.scope "Animated"]
-external _loop: (Animation.t, {. "iterations": int}) => Animation.t = "loop";
+type loopConfig;
+
+[@bs.obj] external loopConfig: (~iterations: int) => loopConfig = "";
 
-let loop = (~iterations=(-1), ~animation, ()) =>
-  _loop(animation, {"iterations": iterations});
+[@bs.module "react-native"] [@bs.scope "Animated"]
+external loop: (Animation.t, loopConfig) => Animation.t = "";
```

Comment:
What I meant was

```re
[@bs.module "react-native"] [@bs.scope "Animated"]
external loop: Animation.t => Animation.t = "loop";

[@bs.module "react-native"] [@bs.scope "Animated"]
external loopWithConfig: (Animation.t, loopConfig) => Animation.t = "loop";
```

Reply:
Sorry read your comment too quickly.

I implemented that change, but making config non-optional might have been cleaner.